src/Darnytsia.Creatio.Core/Features/Books/Handlers/UpdateBookAuthorHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Darnytsia.Creatio.Abstractions;
+using Darnytsia.Creatio.Core.Features.Books.Commands;
+using Darnytsia.Creatio.Data.Entities;
+using Darnytsia.Creatio.Data;
+using System.Linq;
+using System.Threading;
+
+namespace Darnytsia.Creatio.Core.Features.Books.Handlers;
+
+public class UpdateBookAuthorHandler : IRequestHandler<UpdateBookAuthorCommand>
```

Comment:
should be renamed to FillingBookAuthorsHandler
command, query, handler may not always contain insert, get, select, update, delete. Name can be in any understandable form

Reply:
changed